test/drlg_l2_test.cpp

```diff
@@ -50,41 +50,33 @@ void TestCreateL2Dungeon(int level, uint32_t seed, lvl_entry entry)
 TEST(Drlg_l1, CreateL2Dungeon_diablo_5_1677631846)
 {
 	TestCreateL2Dungeon(5, 1677631846, ENTRY_MAIN);
-	EXPECT_EQ(ViewPosition.x, 27);
-	EXPECT_EQ(ViewPosition.y, 28);
+	EXPECT_EQ(ViewPosition, Point(27, 28));
 	TestCreateL2Dungeon(5, 1677631846, ENTRY_PREV);
-	EXPECT_EQ(ViewPosition.x, 26);
-	EXPECT_EQ(ViewPosition.y, 62);
+	EXPECT_EQ(ViewPosition, Point(26, 62));
 }
 
 TEST(Drlg_l1, CreateL2Dungeon_diablo_6_2034738122)
 {
 	TestCreateL2Dungeon(6, 2034738122, ENTRY_MAIN);
-	EXPECT_EQ(ViewPosition.x, 33);
-	EXPECT_EQ(ViewPosition.y, 26);
+	EXPECT_EQ(ViewPosition, Point(33, 26));

```

Comment:
Thanks, I wasn't able to get this working when I first added the tests :)

Reply:
I actually had to look up other tests to see how I'd been comparing points before, because I remember it working 😅 . `(Point {1 ,2})` prevents syntax errors from the macro expansion but it's ugly, so added the constructor to match Size/Displacement